src/Microsoft.ML.TensorFlow/TensorFlow/Tensorflow.cs

```diff
@@ -268,7 +268,7 @@ internal class TFStatus : TFDisposable
     {
         // extern TF_Status * TF_NewStatus ();
         [DllImport(NativeBinding.TensorFlowLibrary)]
-        private static extern unsafe TF_Status TF_NewStatus();
+        internal static extern unsafe TF_Status TF_NewStatus();
```

Comment:
>internal [](start = 8, length = 8)

Instead of exposing the C-API directly, you can use the C# API defined on top of it (the ctor below). Same for the other C imports below.